runtime/jvmti/jvmtiEventManagement.c

```diff
@@ -193,7 +193,7 @@ jvmtiSetEventNotificationMode(jvmtiEnv* env,
 
 		/* Disallow certain events at the thread level */
 
-		switch(event_type) {
+		switch((int)event_type) {
```

Comment:
Is this cast (and the one at L103) required?

Reply:
Yes, see https://github.com/eclipse-openj9/openj9/pull/16501#discussion_r1086689672

Reply:
If you're changing a line, may as well change it properly now. Plans to fix things later all too often get forgotten.